app/frontend/src/pages/chat/Chat.module.css

```diff
@@ -43,11 +44,10 @@
 .chatMessageStream {
     flex-grow: 1;
     max-height: 64rem;
-    max-width: 64.25rem;
     width: 100%;
     overflow-y: auto;
-    padding-left: 1rem;
-    padding-right: 1rem;
+    padding-left: 15vw;
+    padding-right: 15vw;
     display: flex;
     flex-direction: column;
```

Comment:
resize: horizontal; /* Allows horizontal resizing */

Reply:
copy that, resizable achieved